apps/mymedicare_cb/models.py

```diff
@@ -1,4 +1,87 @@
+import requests
+import logging
 from django.db import models
+from django.contrib.auth.models import User, Group
+from apps.accounts.models import UserProfile
+from apps.fhir.authentication import convert_sls_uuid
+from apps.fhir.bluebutton.models import Crosswalk
+from apps.fhir.bluebutton.utils import get_resourcerouter, FhirServerAuth
+
+__author__ = "Alan Viars"
+
+logger = logging.getLogger('hhs_server.%s' % __name__)
+
+
+def get_and_update_user(user_info):
```

Comment:
I wanted to get this logic behind an interface but I'm not convinced this is the best way, I'd appreciate suggestions!